htdocs/comm/remise.php

```diff
@@ -56,9 +56,9 @@
 	$discount_type = GETPOST('discount_type', 'int');
 
 	if (!empty($discount_type)) {
-		$result = $object->set_remise_supplier(price2num(GETPOST("remise")), GETPOST("note", "alphanohtml"), $user);
+		$result = $object->set_remise_supplier(price2num(GETPOST("remise")), GETPOST("note", "san_alpha"), $user);
```

Comment:
The field note here seems to be a "one line text". So recommended value is "alphanohtml".
Why do you need "san_alpha" ?